proto/gossip.proto

```diff
@@ -44,6 +46,61 @@ message GossipRequest {
 message GossipResponse {
   // Requested delta of server's infostore.
   optional bytes delta = 1;
+  // TODO(thschroeter): replace bytes with GossipInfoStore
+  //optional GossipInfoStore delta = 1;
   // Non-nil means client should retry with this address.
   optional Addr alternate = 2;
 }
+
+message GossipInfoStore {
```

Comment:
@tamird, thanks, will do. I've seen a similar approach in `sql/driver/wire.proto`.